x/accesscontrol/types/graph.go

```diff
@@ -257,9 +263,12 @@ func (dag *Dag) getDependencyReads(node DagNode, dependentResource acltypes.Reso
 			// we can add all node IDs as dependencies if applicable
 			nodeIDsMaybeDependency = getAllNodeIDsFromIdentifierMapping(identifierNodeMapping)
 		} else {
-			// TODO: otherwise we need to have partial filtering on identifiers
-			// for now, lets just perform exact matching on identifiers
-			nodeIDsMaybeDependency = identifierNodeMapping[node.AccessOperation.IdentifierTemplate]
+			if node.AccessOperation.IdentifierTemplate != "*" {
```

Comment:
nit: maybe worth adding some testing for * vs identifer for here in `graph_test.go` since it's important to the dependency graph generation.

Reply:
yeah good call, I'll add in some tests for this

Reply:
@eric-zhu-uw I updated this btw, gonna merge the change in if no objections